test_framework/scripts/longhorn-setup.sh

```diff
@@ -114,6 +114,7 @@ wait_longhorn_status_running(){
 
 get_longhorn_manifest(){
   wget ${LONGHORN_MANIFEST_URL} -P ${TF_VAR_tf_workspace}
+  sed -i ':a;N;$!ba;s/---\n---/---/g' "${TF_VAR_tf_workspace}/longhorn.yaml"
```

Comment:
Why we have some chars before the replacement command? Is this intended?

Reply:
sed does its job line by line so if we'd like to make it do this kind of cross-line replacement, we need this trick to let it read all lines of this document and then start the replacement.

ref: https://stackoverflow.com/a/1252191

Reply:
I see. Thanks.